### PR TITLE
PLT-2561 Add commandline option to upload license file

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: mattermost.go [hunk 1]
```diff
@@ -541,6 +547,37 @@ func cmdPermDeleteTeam() {
 	}
 }
 
+func cmdUploadLicense() {
+	if flagCmdUploadLicense {
+		/*if model.BuildEnterpriseReady != "true" {
```
Comment (crspeller) [inline mattermost.go]: Why is this commented out?

Comment (jwilander) [inline mattermost.go]: Oops, forgot to uncomment that

**Comment by crspeller** [comment]
+1

**Comment by hmhealey** [comment]
+1

**Pull request merged**